Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM docker.stackable.tech/stackable/hadoop:3.3.6-stackable0.0.0-dev
+
+COPY --chown=stackable:stackable ./hdfs-group-mapper-0.1.0-SNAPSHOT.jar /stackable/hadoop/share/hadoop/tools/lib/
```

Comment:
This path is not on the classpath by default, I was unsure when adding the rack id generator if we want to just chuck it into a directory thats on the classpath or add it to the classpath explicitly in the operator when it is in use..

Reply:
I think this way is more declarative, rather than just throwing it into the "general" lib folder, but this is just for the test anyway. When this is moved into the docker image I guess we will/could have several jars in that location (topology, group mapper, opa authorizer).

Reply:
Yeah +1 that we should put it the "general" lib folder in the final hadoop image, don't care about this intermediate test state